modular_skyrat/modules/xenoarch/code/modules/research/xenoarch/xenoarch_item.dm

```diff
@@ -2,19 +2,98 @@
+			balloon_alert(user, "stand still!")
+			return
+		balloon_alert(user, "magnified")
+		spawn_magnified(magnified_number)
+		return
+	return ..()
+
+/obj/item/xenoarch/useless_relic/proc/spawn_magnified(type_number)
+	var/obj/item/xenoarch/useless_relic/magnified/new_item = new(get_turf(src))
+	new_item.icon_state = "useless[type_number]"
+	switch(type_number)
+		if(1)
+			new_item.name = "ancient urn"
+			new_item.desc = "This useless relic is an ancient urn that dates from around [rand(400,600)] years ago. \
+			It has made of a ceramic substance and is clearly crumbling at the edges. Perhaps it has ashes \
+			of someone from long ago."
+		if(2)
+			new_item.name = "ancient bowl"
+			new_item.desc = "This useless relic is an ancient bowl that dates from around [rand(400,600)] years ago. \
+			It is made of a bronze alloy and is dented, with some scratches along the inside. Perhaps it could \
+			have had DNA of someone from long ago."
+		if(3)
+			new_item.name = "ancient crown"
+			new_item.desc = "This useless relic is an ancient crown that dates from around [rand(900,1100)] years ago. \
+			It is made from some unknown alloy, with small inlets that would have been used for jewels. Perhaps if we \
+			look around, we could find some of those old jewels."
+		if(4)
+			new_item.name = "ancient coil"
+			new_item.desc = "This useless relic is an ancient coil that dates from around [rand(400,600)] years ago. \
+			It is made of iron and copper. It has some burn marks around the iron rod. Perhaps later on, we could \
+			use it for some machines."
+		if(5)
+			new_item.name = "ancient light"
+			new_item.desc = "This useless relic is an ancient light that dates from around [rand(400,600)] years ago. \
+			It is made of iron and has glass shards around it. It has dents on the iron and clear damage from misuse. \
+			Perhaps we could research this later on to see how the ancients made lights."
+		if(6)
+			new_item.name = "ancient cup"
+			new_item.desc = "This useless relic is an ancient cup that dates from around [rand(900,1100)] years ago. \
+			It is made of hardened stone. There are small cracks all along the surface, as long as chisel marks. \
+			Perhaps it will give insight into the ancient's eating and drinking habits."
+		if(7)
+			new_item.name = "ancient utensils"
+			new_item.desc = "These useless relics are ancient utensils that dates from around [rand(900,1100)] years ago. \
+			It is made of hardened stone. There are small cracks all along the surface, as long as chisel marks. \
+			Perhaps it will give insight into the ancient's eating and drinking habits."
+		if(8)
+			new_item.name = "ancient rock bowl"
+			new_item.desc = "This useless relic is an ancient rock bowl that dates from around [rand(900,1100)] years ago. \
+			It is made of hardened stone. There are small cracks all along the surface, as long as chisel marks. \
+			Perhaps it will give insight into the ancient's eating and drinking habits."
```

Comment:
These numbers could use being defines. The ones in the switch case. Just so it's easier to find what you're looking for.

Reply:
eh, the number is randomly determined by the useless relics initialize. I understand that defining it could be argued as improved code readability, but I would also argue that I use rand(1,8) in the initialize and that by using these numbers, they can see that it corresponds/connects to each other.